tests/framework/base/EventPriorityQueueTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+namespace yiiunit\framework\base;
+
+use yii\base\EventPriorityQueue;
+use yiiunit\TestCase;
+
+/**
+ * @group base
+ */
+class EventPriorityQueueTest extends TestCase
+{
+	/**
+	 * @var EventPriorityQueue
+	 */
+	protected $eventPriorityQueue;
+
+	protected function setUp()
+	{
+		parent::setUp();
+		$this->eventPriorityQueue = new EventPriorityQueue();
```

Comment:
No need to expose the Queue. It should be created for each test.

Create method like `getQueue()` and call it in every test to get a clear instance of `EventPriorityQueue`